src/email/index.js

```diff
@@ -55,9 +55,9 @@ const sendForgotPassword = async (userEmail, resetPasswordToken) => {
   }
 }
 
-const sendContactCheckInEmail = async (info) => {
+const sendContactCheckInEmail = async (info, origin) => {
```

Comment:
could also let `origin = null` here so we can _optionally_ send the fro address with the rest of the request and have it default back to something set in `process.env`. This is also a good opportunity to look at setting system vars into the db somehow. That might be out of scope here though.

Reply:
@revjtanton Updated the PR to have a fallback on env variable for the vue app url